src/config/user.json

```
@@ -1,3 +1,3 @@
 {
-  "username": ""
+  "username": "pranay202"
```

Please revert the changes here. This change will make LinkFree not work.
```suggestion
  "username": ""
```